src/views/Forbidden.vue

```diff
@@ -58,11 +58,18 @@ export default defineComponent({
 .forbidden {
   --timingFunction: cubic-bezier(.785, .135, .15, .86);
   height: calc(100vh - var(--headerHeight));
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 
   h1,
   .circle {
     opacity: 0;
     animation: .75s var(--timingFunction) forwards fadeIn;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
with `h1, .circle` these flex styles would be applied to the `h1` as well - is this intended? If not, we would want to target `.circle` separately.